.github/workflows/publish.yaml

```
@@ -2,10 +2,10 @@ name: Release to PyPI
 
 on:
   release:
-    types: [published]
+    types: [ published ]
 
 jobs:
-  build:
+  publish-pipy:
```

```suggestion
  publish-to-pipy:
```
Maybe like this. Nice find, though.